spec/lib/expired_record_cleaner_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe ExpiredRecordCleaner do
+  describe "fiscal year" do
```

Comment:
The `vaccum_proposal` method is public because I wanted to be able to use it while manually testing in a live environment. I.e. try it on a single proposal before sending email to 100s of people.

There are `expect` statements already around the mailer deliveries for each method. Were you wanting to see something more explicit about the kind of mail, or the content of the mail?